Doc/whatsnew/3.12.rst

```diff
@@ -79,6 +79,7 @@ New Features
 Other Language Changes
 ======================
 
+* XXX The :class:`tempfile.NamedTemporaryFile` has a new optional parameter *delete_on_close* (Contributed by Evgeny Zorin in :issue:`14243`.)
```

Comment:
This should go in Improved Modules -> tempfile
```suggestion
* The :class:`tempfile.NamedTemporaryFile` has a new optional parameter *delete_on_close* (Contributed by Evgeny Zorin in :gh:`22431`.)
```

Reply:
@Ev2geny -- you updated the text but you need to move it to the section I mentioned (see https://github.com/python/cpython/blob/5893b5db98b38b17750c0572c7209774a5034898/Doc/whatsnew/3.9.rst?plain=1#L327-L332) for what the section should look like -- you'll need to add `tempfile` as it's the first improvement listed in whatsnew.

A

Reply:
@AA-Turner , thanks. i have fixed it now.